app/src/main/java/org/oppia/app/home/HomeActivityPresenter.kt

```diff
@@ -9,7 +9,7 @@ import org.oppia.app.activity.ActivityScope
 import org.oppia.app.drawer.NavigationDrawerFragment
 import javax.inject.Inject
 
-const val TAG_HOME_FRAGMENT = "HOME_FRAGMENT"
+const val TAG_HOME_FRAGMENT = "HomeFragment.tag"
```

Comment:
Shift to companion object on `HomeFragment`.

Reply:
Also please check this comment:
https://github.com/oppia/oppia-android/pull/1717#discussion_r478456256

As per this comment it should be named like this:
`internal const val TAG = "HomeFragment.tag"`

Reply:
done
But am not sure the way I did it. So please check it and guide me if am wrong in this.